elide-model-config/src/test/resources/validator/table_sql_both_inheritance/models/tables/player_stats_extends.hjson

```diff
@@ -0,0 +1,36 @@
+{
+  tables:
+  [
+    {
+      name: PlayerStatsChild
+      extend: PlayerStats
```

Comment:
Don't we already have an extends test?  I'm confused why we needed to create so many new HJSON test files.

Reply:
These tests are the same as those, but I wanted to have a test case specifically testing the infinite loop and issues related to that in case someone edits those validation methods and test files to work differently.